### PR TITLE
Convert to maven pom

### PR DESCRIPTION
**Pull request by snocke**

### CODE REVIEW

**Comment by snocke** [comment]
https://ci.stackable.tech/view/99%20Dev%20'n%20Demo/job/Functional-Cluster-tests/83/console

**Comment by sbernauer** [comment]
Hi @snocke, looks good overall!

The test fails on my machine with:
```
case.go:363: resource StatefulSet:kuttl-test-modest-gecko/hdfs-journalnode-default: .status.readyReplicas: value mismatch, expected: 3 != actual: 1
[...]
case.go:363: resource StatefulSet:kuttl-test-modest-gecko/hdfs-namenode-default: .status.readyReplicas: value mismatch, expected: 2 != actual: 1
```
After fixing it (see suggestion in this PR) i run into the same problem as yesterday
```
    logger.go:42: 10:43:10 | smoke/6- | + kubectl exec -n kuttl-test-classic-elephant spark-master-default-0 -- /bin/bash -x -v -c /tmp/spark-submit.sh
    logger.go:42: 10:43:16 | smoke/6- | running command: [./tests/kubectl-assert local-file-exist _SUCCESS -v]
    logger.go:42: 10:43:16 | smoke/6- | ASSERT File with name _SUCCESS should be on path.
    logger.go:42: 10:43:16 | smoke/6- | INFO   kubectl -n kuttl-test-classic-elephant get pods -o=name
    logger.go:42: 10:43:16 | smoke/6- | 
    logger.go:42: 10:43:16 | smoke/6- | INFO   kubectl exec -n kuttl-test-classic-elephant -- ls /tmp/stacky
    logger.go:42: 10:43:16 | smoke/6- | error: pod, type/name or --filename must be specified
    logger.go:42: 10:43:16 | smoke/6- | 
    logger.go:42: 10:43:16 | smoke/6- | ASSERT FAIL  does not exist on .
```

Some minor suggestions:
* Maybe we should rename the test folder to something like `smoke-spark-standalone-hdfs` so that we have space for future tests
* We could update the Spark version (is 3.0.1) at the moment. Even better of course would be to try out all the avaiable versions with a matrix ;)
* Is the webhdfs really needed or can we use some HDFS pod and the `hdfs dfs` command?
* It would be very godd if we could assert that some actual data was written (i think the current check looks for the `_SUCCESS` marker file). Especially as in a previous revision the `_SUCCESS` marker file was written but no actual data (for whatever reason)

File: tests/kuttl/smoke/01-assert-hdfs.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  readyReplicas: 2
+  replicas: 2
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: hdfs-journalnode-default
+status:
+  readyReplicas: 3
+  replicas: 3
```
Comment (sbernauer) [inline tests/kuttl/smoke/01-assert-hdfs.yaml:21]:
```suggestion
  readyReplicas: 1
  replicas: 1
```

**Comment by snocke** [comment]
Jenkins run green
https://ci.stackable.tech/view/99%20Dev%20'n%20Demo/job/Functional-Cluster-tests/85/console

**Comment by snocke** [comment]
Hi @sbernauer ,

1. Yes, sounds good
2. Totally agree. Hope to make that happen in the future
3. Not really. That could be changed. It is still a functionality test. However, it is also done in the integration test. Therefore, it is redundant and a different approach could increase test coverage.
4. Agree. However, for the spark standalone I won't put any more work in it because the spark k8s operator is around the corner. We'll see how it works with the new operator I guess. 

I'll put your thoughts in tickets.

**Comment by sbernauer** [comment]
Thanks for your feedback!
Currently the test fails on my machine with a fresh kind cluster (see above), i have to look into it what's going wrong.
Regarding the points:

1. Could you please rename the folder?
2. I would prefer to at least update the Spark version to 3.2.1 (current latest). Should be very easy i guess ^^
3. I personally don't like the webhdfs (complexity and most users would do it with hdfs dfs) but I'm fine too leave it in for now and create a ticket for removal.
4. Hmm i think a test that does not assert that any data was written is not wort so much (especially as we run into the case the `__SUCCESS` exists but the job did not work). Could you please at least check that a `*.csv` file exists?

**Comment by snocke** [comment]
hi @sbernauer,

1. renamed Folders
2. tested with the latest available version 3.1.1. https://ci.stackable.tech/view/99%20Dev%20'n%20Demo/job/Functional-Cluster-tests/86/console
3. ticket created: https://github.com/stackabletech/stackable-cluster/issues/43
4. as explained it is only the case for spark-standalone. The problem will be further investigated with the new spark k8s operator and/or how we're going to proceed with the spark-standalone operator.

**Review by sbernauer — APPROVED** [review]

**Pull request merged**